### ISSUE TITLE
How did you split dataset for training and validation?

### ISSUE THREAD

**Issue by quqixun** [sstzal/DiffTalk#18]
In data dir, there are [data_test.txt](https://github.com/sstzal/DiffTalk/blob/b49a607aa39bb41da215870c2b3dd48e9af9f30d/data/data_test.txt) for validation and [data_train.txt](https://github.com/sstzal/DiffTalk/blob/b49a607aa39bb41da215870c2b3dd48e9af9f30d/data/data_train.txt) for training.
How did you split dataset? By portrait or by videos?
By portrait means persons in training set are not repeated in validation set.
By videos means randomly spit all videos into  training and validation set.

In [train_name.txt](https://github.com/sstzal/DiffTalk/blob/b49a607aa39bb41da215870c2b3dd48e9af9f30d/data/train_name.txt), there are 98 videos. However, 99 videos can be found in [data_train.txt](https://github.com/sstzal/DiffTalk/blob/b49a607aa39bb41da215870c2b3dd48e9af9f30d/data/data_train.txt).
What is the relationship between train_name.txt and data_train.txt?

Could you provide test_name.txt just like train_name.txt to indicate videos used in validation?
